src/pages/en/guides/data-fetching.md

```diff
@@ -4,35 +4,109 @@ title: Data Fetching
 description: Learn how to fetch remote data with Astro using the fetch API.
 ---
 
-Astro components and pages can fetch remote data to help generate your pages. Astro provides two different tools to pages to help you do this: **fetch()** and **top-level await.**
+Astro pages can fetch remote data at build time to help generate your pages.
```

Comment:
Mention components can also fetch.

```suggestion
Astro pages and components can fetch remote data at build time to help generate your site.
```

Reply:
See comment below. I think ONE of these will become `.astro` files, and the other will be Astro components. Might even just come down to which visually strikes me better.